### ISSUE TITLE
Add Fluid Transposer recipes for elemental powders

### ISSUE THREAD

**Issue by CyricV** [CyricV/TelluraPack#1]
Fluid Transposer seems to have an interesting error.

Anyway, I have a more interesting issue that's been trying me this morning. I'm creating Fluid Transposer recipes for Blaze, Blizz, Blitz, and Basalz Powders using liquid redstone and Fire, Water, Air, and Earth essences from Magical Crops. The problem is the recipes are seeing each other as duplicates. Here is the code I'm using:

[CODE]
var fireEssence                 = magicalcrops:magicalcrops_FireEssence;
var waterEssence                = magicalcrops:magicalcrops_WaterEssence;
var airEssence                  = magicalcrops:magicalcrops_AirEssence;
var earthEssence                = magicalcrops:magicalcrops_EarthEssence;
# Blaze Powder

recipes.remove(blazePowder);
mods.thermalexpansion.Transposer.removeFillRecipe(glowstoneDust, liquid:redstone);
mods.thermalexpansion.Transposer.addFillRecipe(2000, fireEssence, blazePowder \* 2, liquid:redstone \* 100);
# Blizz Powder

recipes.remove(blizzPowder);
mods.thermalexpansion.Transposer.removeFillRecipe(snowball, liquid:redstone);
mods.thermalexpansion.Transposer.addFillRecipe(2000, waterEssence, blizzPowder \* 2, liquid:redstone \* 100);
# Blitz Powder

recipes.remove(blitzPowder);
mods.thermalexpansion.Transposer.removeFillRecipe(sand, liquid:redstone);
mods.thermalexpansion.Transposer.addFillRecipe(2000, airEssence, blitzPowder \* 2, liquid:redstone \* 100);
# Basalz Powder

recipes.remove(basalzPowder);
mods.thermalexpansion.Transposer.removeFillRecipe(allObsidianDust, liquid:redstone);
mods.thermalexpansion.Transposer.addFillRecipe(2000, earthEssence, basalzPowder \* 2, liquid:redstone \* 100);
[/CODE]

[SPOILER="Warning Messages"]

[12:24:50] [Server thread/INFO]: CyricV joined the game
[12:24:52] [Client thread/INFO]: [CHAT] WARNING: [ModTweaker2] Duplicate Thermal Expansion Transposer (Fill) Recipe found for magicalcrops:magicalcrops_WaterEssence and liquid:redstone \* 100. Command ignored!
[12:24:52] [Client thread/INFO]: [CHAT] WARNING: [ModTweaker2] Duplicate Thermal Expansion Transposer (Fill) Recipe found for magicalcrops:magicalcrops_AirEssence and liquid:redstone \* 100. Command ignored!
[12:24:52] [Client thread/INFO]: [CHAT] WARNING: [ModTweaker2] Duplicate Thermal Expansion Transposer (Fill) Recipe found for magicalcrops:magicalcrops_EarthEssence and liquid:redstone \* 100. Command ignored!
[/SPOILER]

It seems all the essences I'm using are in an oredict by the name of accioMaterials. When I remove them from the oredict with the following code I get some interesting behaviors.

[CODE]
accioMaterial.remove(fireEssence);
accioMaterial.remove(waterEssence);
accioMaterial.remove(airEssence);
accioMaterial.remove(earthEssence);
[/CODE]

On initial load of Minecraft I receive the same warnings about duplicate recipes, however if I switch at least three of the recipes to take anything not in accioMaterial (tested by using essence seeds because I could just change one word in each of the variable definitions) and reload the scripts it works fine. Then I switch back to the correct recipes using the essences and reload once again they all work.

This leads me to believe it's a load order issue, but I don't know how to enforce it so the ordict entries are removed first other than alphabetical order with file names.

In summary, the Fluid Transposer is seeing several items with different item IDs as the same thing and I don't know why.
